airbyte-ci/connectors/pipelines/pyproject.toml

```diff
@@ -25,10 +25,12 @@ connector-ops = {path = "../connector_ops", develop = true}
 toml = "^0.10.2"
 sentry-sdk = "^1.28.1"
 segment-analytics-python = "^2.2.3"
+asyncclick = "^8.1.3.4"
 
 [tool.poetry.group.test.dependencies]
-pytest = "^6.2.5"
+pytest = "^7.0.0"
```

Comment:
This bump was required in order to be able to use `pytest.asyncio`. Confirmed that all tests still run and pass after this